app/src/ui/history/commit-list-item.tsx

```diff
@@ -227,7 +239,25 @@ export class CommitListItem extends React.PureComponent<
       }
     )
 
-    showContextualMenu(items)
+    return items
+  }
+
+  private getContextMenuMultipleCommits(): IMenuItem[] {
+    const items: IMenuItem[] = []
+
+    const count = this.props.selectedCommits.length
+    const pluralize = count === 1 ? '' : 's'
+    const countOut = count === 1 ? '' : ` ${count} `
+    if (enableCherryPicking()) {
+      items.push({
+        label: __DARWIN__
+          ? `Cherry Pick${countOut}Commit${pluralize}…`
+          : `Cherry pick${countOut}commit${pluralize}…`,
+        action: this.onCherryPick,
+      })
+    }
```

Comment:
Unless I'm missing something, we shouldn't need to check the `count` here, right? In this method there will always be `count > 1` commits. Otherwise, I think would have a weird string with `count === 1` unless we add a space to the `countOut` string.
```suggestion
    const pluralize = count === 1 ? '' : 's'
    const countOut = count === 1 ? ' ' : ` ${count} `
    if (enableCherryPicking()) {
      items.push({
        label: __DARWIN__
          ? `Cherry Pick${countOut}Commit${pluralize}…`
          : `Cherry pick${countOut}commit${pluralize}…`,
        action: this.onCherryPick,
      })
    }
```

Reply:
Absolutely.. good catch - from before I split it out into to methods.